README.md

```diff
@@ -53,10 +53,44 @@ We are using continuous integration software applied to the supplied test datase
 
 ### Folder Structure
 
+Our folder structure is designed to separate each analysis into its own set of notebooks that are independent of other analyses.
+Within the analyses directory, create a folder for your analysis.
+Choose a name that is unique from other analyses and somewhat detailed.
+For example, instead of `gene-expression`, choose `gene-expression-clustering` if you are clustering samples by their gene expression values.
+You should assume that any data files that you need to read live in the `../../data` directory and that their file names match what the `data_downloader.sh` script produces.
+These files should be read in at their relative path, so that we can re-run analyses if the underlying data change.
+
+```
+OpenPBTA-analysis
+├── CONTRIBUTING.md
+├── README.md
+├── analyses
+│   ├── existing-analysis-1
+│   ├── existing-analysis-2
+│   └── new-analysis
+│       ├── 00.run_all.sh
+│       ├── 01.preprocess-data.Rmd
```

Comment:
Should it be emphasized somewhere when a `.Rmd` vs `.R` should be used or do we just want people to always heavily default to `.Rmd`?

Reply:
Added additional info that says this is an example, but other formats are fine 👍